docs/index.md

```diff
@@ -59,12 +59,22 @@ to interact with Tachyon, e.g. copy data in and out of the file system.
 # User Documentation
 
 **Deployment Guide:**
-
+<!---
```

Comment:
Actually, I don't think we should modify much of this main page for now. That means we don't need all the new icons and we don't need the new links in this index page. These pages should be purely documentation for now.

We can add the new links/images later. Thanks!